io.github.zen_browser.app.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.zen_browser.app
```

Comment:
app is a bad id, use `io.github.zen_browser.zen`. You need to create a repo called `zen` https://github.com/zen-browser/ here or rename the `desktop` repo to `zen`.

Reply:
can't I put "desktop" instead of app?

Reply:
No, `desktop` is blocked. https://docs.flathub.org/docs/for-app-authors/requirements#application-id see the first bullet point.

Reply:
Oh 😅. Sorry. Does the "zen" repo have to contain any contents?

Reply:
No

Reply:
Once I fix this, would everything be alright?

Reply:
> Once I fix this, would everything be alright?

You should move metainfo, icon and desktop file to a repo there and remove them from here.